config/locales/mailers/en.yml

```diff
@@ -54,9 +54,12 @@ en:
   user_mailer:
     invite_increase_notification:
       subject: "[%{app_name}] New invitations"
-      body:
-        one: "We just wanted to let you know that you have %{count} new invitation, which can be used to create a new account at the Archive. You can invite a friend at %{invitation_page}."
-        other: "We just wanted to let you know that you have %{count} new invitations, which can be used to create new accounts at the Archive. You can invite a friend at %{invitation_page}."
+      html_body:
```

Comment:
The text changes look good to me.

For other reviewers: are we fine with naming the keys like this, or should we use nesting? Something like:
```yaml
html:
  body:
    one:
    other:
text:
  body:
    one:
    other:
```

Reply:
I think nesting, since it's our standard.